docs/sphinx/examples/cpp/dynamics/heisenberg_model.cpp

```diff
@@ -44,7 +44,7 @@ int main() {
   // The staggered magnetization operator is used to measure antiferromagnetic
   // order. It is defined as a sum over all spins of the Z operator, alternating
   // in sign. For even sites, we add `sz`; for odd sites, we subtract `sz`.
-  auto staggered_magnetization_t = cudaq::sum_op<cudaq::spin_handler>::empty();
+  auto staggered_magnetization_t = cudaq::sum_op<cudaq::spin_op>::empty();
```

Comment:
```suggestion
  auto staggered_magnetization_t = cudaq::spin_op::empty();
```
As written this won't work. Did you try running the examples? When you change examples, it is good to manually check that they all run successfully, since validation for them is not necessarily happening in CI.